src/components/pages/CommentList.vue

```diff
@@ -1,107 +1,123 @@
 <template>
 	<div class="comment-list">
 		<h1 class="title">{{ $t( 'donation_comments_title' ) }}</h1>
-		<span>{{ $t( 'donation_comments_text' )}}</span>
+		<span>{{ $t( 'donation_comments_text' ) }}</span>
 		<span v-if="isLoading" class="has-margin-top-36 columns is-centered">
-			<span class="pseudo button is-loading"></span>
-		</span>
+      <span class="pseudo button is-loading"></span>
+    </span>
 		<div class="has-margin-top-18">
-			<div class="has-margin-top-18" v-for="comment in pageContent">
-				<div class="has-text-weight-bold">{{ $t( 'donation_comments_donor_headline', headlineVars( comment ) )}}</div>
+			<div class="has-margin-top-18" v-for="comment in pageContent" :key="comment.id">
+				<div class="has-text-weight-bold">{{
+						$t( 'donation_comments_donor_headline', headlineVars( comment ) )
+					}}
+				</div>
 				<div class="has-text-gray-dark">{{ comment.date }}</div>
 				<div>{{ comment.comment }}</div>
 			</div>
 		</div>
 		<div v-if="!isLoading" class="page-selector has-margin-top-36 has-margin-bottom-18">
-			<a class="button mdi mdi-arrow-left" :disabled="currentPage === 1" v-on:click="previousPage"></a>
+			<a class="button mdi mdi-arrow-left" :disabled="currentPage === 1" @click="previousPage"></a>
 			<FunSelect class="is-form-input" v-model="currentPage" name="page" select-id="page">
-				<option v-for="page in pageCount" :value="page"> {{ page }}</option>
+				<option v-for="page in pageCount" :value="page">{{ page }}</option>
 			</FunSelect>
-			<a class="button mdi mdi-arrow-right" :disabled="currentPage === pageCount" v-on:click="nextPage"></a>
+			<a class="button mdi mdi-arrow-right" :disabled="currentPage === pageCount" @click="nextPage"></a>
 		</div>
 	</div>
 </template>
 
-<script lang="ts">
-import { defineComponent } from 'vue';
+<script setup lang="ts">
 import axios from 'axios';
 import { commentModelsFromObject } from '@src/view_models/Comment';
-import FunSelect from '@src/components/shared/legacy_form_inputs/FunSelect.vue';
 
 const PAGE_SIZE = 10;
 
+const comments = ref( [] );
+let pageContent = ref( [] );
```

Comment:
`comments` and `pageContent` could be improved to be `ref<Comment[]>([])`
The `Comment` interface can be imported from `'@src/view_models/Comment'`